canvas/cli/assignment/fetch.py

```diff
@@ -5,12 +5,11 @@
 import canvas.cli.common
 import canvas.config
 
-DEFAULT_TABLE = False
 DEFAULT_SKIP_HEADERS = False
```

Comment:
Seems like `--skip-headers` can probably be treated the same way as `--table`,
and be put in a common location (and added as a part of canvas.cli.common.add_output_args()).

Go ahead and move it if this makes sense.